react/week3/todo3/src/Components/TodoForm.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+class TodoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { todoInput: "" };
+  }
+
+  handleChange = (event) => {
+    this.setState({ todoInput: event.target.value });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.props.addTodo(this.state.todoInput);
+  };
+
+  render() {
+    return (
+      <form onSubmit={(event) => this.handleSubmit(event)}>
```

Comment:
When you're passing the same parameter to both methods like here, you can use the shorthand:
`onSubmit={this.handleSubmit}`